boilerplate.typescript.js

```diff
@@ -17,4 +17,5 @@ exports.getPackageTemplates = (packageRelativePath) => [
   { template: 'packages/.yarnrc.ejs', target: `${packageRelativePath}/.yarnrc` },
   { template: 'packages/.npmignore.ejs', target: `${packageRelativePath}/.npmignore` },
   { template: 'packages/package.json.ejs', target: `${packageRelativePath}/package.json` },
+  { template: 'packages/tsconfig.json.ejs', target: `${packageRelativePath}/tsconfig.json` },
```

Comment:
If you're not using any EJS templates in `boilerplate/typescript/packages/tsconfig.json.ejs`, the rename was not necessary. Original `tsconfig.json` file is copied in https://github.com/technology-studio/ignite-txo-package-boilerplate/blob/3ee375e34f68163f8a5d6c5c9dd8e92cb26dbad1/boilerplate.js#L106-L109